.github/stale.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 7
```

Comment:
I think 7 is a bit low, should we do 14 or 21? We do not very actively look at issues and this will give us enough time to respond.